docs/how-to/kafka-connect.md

```diff
@@ -11,14 +11,14 @@ This How-to guide covers deploying Kafka Connect, integrating it with Charmed Ap
 
 ## Prerequisites
 
-For this guide, we will need an active Charmed Apache Kafka application, either using Apache ZooKeeper or in KRaft mode. Follow the [How to deploy Charmed Apache Kafka](https://discourse.charmhub.io/t/charmed-kafka-documentation-how-to-deploy/13261) guide to set up the environment.
+For this guide, we will need an active Charmed Apache Kafka application. Follow the [How to deploy Charmed Apache Kafka](https://discourse.charmhub.io/t/charmed-kafka-documentation-how-to-deploy/13261) guide to set up the environment.
 
 ## Deploy and set up
 
 To deploy [Kafka Connect charm](https://charmhub.io/kafka-connect) and integrate it with Charmed Apache Kafka, use the following commands:
 
 ```bash
-juju deploy kafka-connect --channel latest/edge
+juju deploy kafka-connect --channel /edge
```

Comment:
Question: Do we need something instead of latest here?

Reply:
Ah good catch, just `edge`. We update docs when it's pushed to higher tiers.